qgreenland/config/layers/Geophysics/Albedo/albedo.py

```diff
@@ -0,0 +1,34 @@
+from qgreenland.config.datasets.albedo import (
+    monthly_albedo as dataset,
+)
+from qgreenland.config.helpers.steps.compress_and_add_overviews import (
+    compress_and_add_overviews,
+)
+from qgreenland.models.config.layer import Layer, LayerInput
+
+
+layers = [Layer(
+    id=f'albedo_{year}_07',
+    title=f'July {year} albedo (1km)',
+    description=(
+        f"""Monthly average broadband planar albedo for July {year} derived from
```

Comment:
"Monthly average... for July {year}"

"Monthly" seems redundant here. What about "Average ... for July {year}"?

Reply:
Good catch!

Reply:
2bdee06e